src/runtime/internal/lifecycle.ts

```diff
@@ -151,6 +149,25 @@ export function bubble(component, event) {
 
 	if (callbacks) {
 		// @ts-ignore
-		callbacks.slice().forEach(fn => fn.call(this, event));
+		invoke_callbacks(callbacks, this, event);
 	}
 }
+
+// Invoke all callbacks, even if an error occur
+function invoke_callbacks(callbacks, source, event) {
+	callbacks.slice().forEach(fn => {
+		try {
+			fn.call(source, event);
+		} catch (err) {
+			// @ts-ignore
+			if (window.reportError) {
```

Comment:
question for more knowledgeable reviewers: will this file be called during ssr? if so, we probably need to fix this reference to `window`. also note that `reportError` is not yet supported in Node

Reply:
Oh I miss that component events can be triggered in SSR !
Don't know how to handle this correctly...